internal/openshift/client/openshift_client.go

```diff
@@ -14,6 +14,7 @@ import (
 
 	"github.com/fabric8-services/fabric8-jenkins-idler/internal/model"
 	log "github.com/sirupsen/logrus"
+	"k8s.io/api/core/v1"
```

Comment:
Did you look at using an openshift client instead?

Reply:
You will need this 
```patch
diff --git a/Gopkg.toml b/Gopkg.toml
index d6a8d93..523442b 100644
--- a/Gopkg.toml
+++ b/Gopkg.toml
@@ -10,6 +10,11 @@
   name = "github.com/sirupsen/logrus"
   version = "=v1.0.0"
 
+[[constraint]]
+  name = "k8s.io/api"
+  source = "github.com/openshift/kubernetes-api"
+  version = "kubernetes-1.10.2"
+
 [[constraint]]
   name = "github.com/goadesign/goa"
   version = "=v1.3.1"
```
and the updated lock file.

Reply:
why should source be `github.com/openshift/kubernetes-api` ? I see that this is a fork of `k8s.io/api` and is not maintained from a little less than a year

Reply:
> is not maintained from a little less than a year

Really ! I am curious how you arrived at that conclusion . I arrived at a different conclusion after looking at - https://github.com/openshift/kubernetes-api/releases

Reply:
I went with that repo because that is what is being used in OSO  - https://github.com/openshift/origin/blob/db3845c60123e1ad62793307aa689b9711dd364c/glide.yaml#L17-L19

Reply:
Cool. 
github.com/openshift/kubernetes-api 
is more appropriate then

Reply:
> I arrived at a different conclusion after looking at

Last commit was in October, 2017

Reply:
Shouldn't version be `version = "=origin-3.10-kubernetes-1.10.2"`

Reply:
wow, I can't import this thing as `github.com/openshift/kubernetes-api/core/v1`, I will have to import it as `k8s.io/api/core/v1`